openedx_events/tooling.py

```diff
@@ -185,6 +151,81 @@ def validate_sender():
 
         return responses
 
+    def send_event(self, send_robust=True, time=None, **kwargs):
```

Comment:
Unfortunately, time is optional and after send_robust, which is the opposite order of the arguments in the new method, to keep this as backward compatible as possible.

Reply:
@ormsbee: FYI: Are you ok with this to avoid a breaking change? I know you wish time was required, and if we don't do that now, we probably will never do that.

Reply:
Note: I plan to merge this. We can always refactor later. Clearly, the sooner the better, if we plan to break things.